hvpy/facade.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from hvpy.core import execute_api_call
+from hvpy.parameters import getJP2ImageInputParameters
+
+__all__ = ["getJP2Image"]
+
+
+def copy_docstring(input_class):
+    def decorator(func):
+        func.__doc__ = func.__doc__ + input_class.__doc__
+        return func
+
+    return decorator
+
+
+@copy_docstring(getJP2ImageInputParameters)
+def getJP2Image(
+    *,
+    date: datetime,
```

Comment:
Get this when keyword arg is not set to `none`
```py
TypeError: getJP2Image() missing 1 required keyword-only argument: 'date'
```
Get this when Keyword args are set to `none`
```py
 E   pydantic.error_wrappers.ValidationError: 1 validation error for getJP2ImageInputParameters
 E   date
 E     none is not an allowed value (type=type_error.none.not_allowed)
```

Reply:
How are you calling the function?

Reply:
> How are you calling the function?

I haven't added the test for errors, but I am doing it like the test here: https://github.com/Helioviewer-Project/python-api/pull/33/files#diff-02727b06210d4e69cdfb598c976fb52dd3adddfa7c7f1f48c98a83f36b8f1db4R6

Reply:
Oh right, I can't read. I think it might be easier to drop them being keywords only.